pkgs/development/compilers/go/1.19.nix

```
@@ -17,7 +17,7 @@
 }:
 
 let
-  goBootstrap = buildPackages.callPackage ./bootstrap116.nix { };
+  goBootstrap = if stdenv.buildPlatform.isMusl then buildPackages.gccgo else "${buildPackages.callPackage ./bootstrap116.nix {}}/share/go";
```

Changing causes the mass rebuild and breaks the non-musl bootstrap, it's also used as passthru and disallowedReferences. Easier to leave it as it is.
```suggestion
  goBootstrap = if stdenv.buildPlatform.isMusl then buildPackages.gccgo else buildPackages.callPackage ./bootstrap116.nix { };
```

---

I could not figure out why it caused a mass-rebuild. Thanks!